controller_manager:
  ros__parameters:
    update_rate: 100

    position_trajectory_controller:
      type: joint_trajectory_controller/JointTrajectoryController

    joint_state_broadcaster:
      type: joint_state_broadcaster/JointStateBroadcaster

position_trajectory_controller:
  ros__parameters:
    command_interfaces:
      - position
    state_interfaces:
      - position
      - velocity
    joints:
      - wrist_bevel_joint
      - wrist_carrier_joint

joint_state_broadcaster:
  ros__parameters:
    joints:
      - carrier_joint
      - bevel_joint